ingestion/src/metadata/ingestion/source/database/bigquery/connection.py

```diff
@@ -74,8 +81,60 @@ def get_connection(connection: BigQueryConnection) -> Engine:
     )
 
 
-def test_connection(engine: Engine) -> None:
+def test_connection(engine: Engine, service_connection) -> str:
```

Comment:
is this the correct type? Or are we returning a `TestStepResult`?

Reply:
It should not be str, i will change this too.